proposals/0311-task-locals.md

```diff
@@ -114,7 +114,7 @@ enum MyLibrary {
 
 Each task-local declaration represents its own, independent task-local storage. Reading from one declaration will never observe a value stored using a different declaration, even if the declarations look exactly the same.
 
-Because of those pitfals with creating multiple instances of the same task local identifier, we propose to diagnose and and fail at compile time unless the `@TaskLocal` property wrapper is defined on something other than a static or global property. 
```

Comment:
The double negative in this paragraph is difficult to decode, and it's unclear if intended or not.  By removing both negatives the meaning becomes clear.